Add ability to delete imposter requests

lucasbode opened the pull request:
In a recent mountebank release they added the ability to delete the saved requests for an imposter - https://www.mbtest.org/releases/v2.2.1

This is helpful for a variety of reasons. For me personally, I use an imposter to mock a messaging queue locally. And between tests I would like to "clear message from my queue" (delete all the requests to an imposter). So having that functionality available in the client would be very helpful.

lucasbode committed:
Merge pull request #1 from mattherman/master

Merge from master.

Commit:
Added method to client to allow for deletion of saved requests for an imposter.

mattherman commented:
Thanks for contributing again. The changes looked good. I should be able to merge it tomorrow morning and get a new package published.

mattherman commented:
I added the client method to the interface (#83) and then published new packages. There is a pre-release package, [v.5.0.0-rc5](https://www.nuget.org/packages/MbDotNet/5.0.0-rc5), that has your changes as-is. I also published a stable package, [v.4.3.2](https://www.nuget.org/packages/MbDotNet/4.3.2), that has the same changes but non-async. This is because I introduced the async methods in v5 since they were breaking changes. I have a few more updates I want to make before publishing a stable v5 version, but I'll hopefully get to that in the near future.

Let me know if you have any questions.

lucasbode commented:
Awesome, thanks for getting this released right away. I plan on pulling it
into my project today!

Thanks,
Lucas

On Thu, Jul 30, 2020 at 11:17 AM Matthew Herman <notifications@github.com>
wrote:

> I added the client method to the interface (#83
> <https://github.com/mattherman/MbDotNet/pull/83>) and then published new
> packages. There is a pre-release package, v.5.0.0-rc5
> <https://www.nuget.org/packages/MbDotNet/5.0.0-rc5>, that has your
> changes as-is. I also published a stable package, v.4.3.2
> <https://www.nuget.org/packages/MbDotNet/4.3.2>, that has the same
> changes but non-async. This is because I introduced the async methods in v5
> since they were breaking changes. I have a few more updates I want to make
> before publishing a stable v5 version, but I'll hopefully get to that in
> the near future.
>
> Let me know if you have any questions.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/mattherman/MbDotNet/pull/82#issuecomment-666500369>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AC4H2I2UQJU5UCIRFBDJVCLR6GMJLANCNFSM4PK5ZC5Q>
> .
>
